pkg/instrumentation/tracing/sampler.go

```diff
@@ -0,0 +1,65 @@
+// Copyright The NRI Plugins Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracing
+
+import (
+	"sync"
+
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+var (
+	drop = sdktrace.SamplingResult{
+		Decision:   sdktrace.Drop,
+		Tracestate: trace.TraceState{},
+	}
+
+	_ sdktrace.Sampler = &sampler{}
+)
+
+type sampler struct {
+	sync.RWMutex // TODO(klihub): should we use an atomic type instead ?
```

Comment:
Are you sure we want to keep // TODO in the code base? The comment looks more like a consideration than something that needs to be done.

Reply:
Well, I wanted to leave that in for myself for reconsideration... but removed it now.